workflows/colab.yml

```
@@ -0,0 +1,25 @@
+name: colab
+on: 
+  push:
+    branches:
+      - master
+jobs:
+  badges:
+    name: Colab Badge
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v2
```

Maybe use newer version here?
```suggestion
        uses: actions/checkout@v3
```